route-rs-runtime/src/link/mod.rs

```diff
@@ -34,9 +34,9 @@ pub use self::blackhole_link::*;
 use crate::element::Element;
 
 /// All Links communicate through streams of packets. This allows them to be composable.
-pub type PacketStream<Input> = Box<dyn Stream<Item = Input, Error = ()> + 'static + Send>;
+pub type PacketStream<Input> = Box<dyn Stream<Item = Input, Error = ()> + Send>;
 /// Some Links may need to be driven by Tokio. This represents a handle to something Tokio can run.
-pub type TokioRunnable = Box<dyn Future<Item = (), Error = ()> + 'static + Send>;
+pub type TokioRunnable = Box<dyn Future<Item = (), Error = ()> + Send>;
```

Comment:
Is removing the static lifetime on these types intentional?

Reply:
Sorta, patched this.